torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -4323,11 +4324,38 @@ void CompilationUnit::define_interface(
     arguments.insert(
         arguments.end(), schema.arguments().begin(), schema.arguments().end());
     iface->addMethod(schema.cloneWithArguments(std::move(arguments)));
-    if (method_def.statements().size() != 1 ||
-        method_def.statements()[0].kind() != TK_PASS) {
+
+    auto stmts_size = method_def.statements().size();
+    // if there is only one element, it should be "pass" statement
+    if (stmts_size == 1 && method_def.statements()[0].kind() != TK_PASS) {
```

Comment:
If seems this loop handles only a special case of the following loop? maybe it can be omitted.

Reply:
oh true. will fix it